index.html

```diff
@@ -0,0 +1,134 @@
+      <p class="venue">@ Activision, Sweden</p>
+    </section>
+    <section class="program" id="program">
+      <div class="box">
+        <h3 class="main_program">Main Program</h3>
+      </div>
+      <ul class="activities">
+        <li class="activity"><img class="img_activity" src="./pics/speech.png" alt="schedule icon">
+          <p class="description">Schedule</p>
+          <p class="explanation">Listen to the communication involving the activities and the rules and regulations for
+            the matches. We shall also talk about the order of the tasks</p>
+        </li>
+        <li class="activity"><img class="img_activity" src="./pics/exhibit.png" alt="exhibit icon">
+          <p class="description">Exhibition</p>
+          <p class="explanation">Take a look at some of the greatest activision has to offer as well as history of
+            activision. View our gallery of older games as well as past achievements</p>
+        </li>
+        <li class="activity"><img class="img_activity" src="./pics/social.png" alt="social icon">
+          <p class="description">Social</p>
+          <p class="explanation">Move around and find new members for your squad and long term gaming partners. You may
+            even get contracts with awesome gaminng crews such as optic gaming</p>
+        </li>
+        <li class="activity"><img class="img_activity" src="./pics/game.png" alt="controllericon">
+          <p class="description">Tournament</p>
+          <p class="explanation">The matches begin. The different teams shall be in their gaming stations and begin the
+            battle for the best. </p>
+        </li>
+        <li class="activity"><img class="img_activity" src="./pics/trophy.png" alt="trophy icon">
+          <p class="description">Winners</p>
+          <p class="explanation">We shall be declaring the winner and handing in prices for best teams. Other prizes
+            will be given as well such as player with most K/D awards, most kills award and so on</p>
+        </li>
+      </ul>
+      <div class="position_url">
+        <a class="whole_program" href="#">SEE THE WHOLE PROGRAM</a>
+      </div>
+      <div class="center_button">
+        <button type="button" class="join">Join warzone tournament</button>
+      </div>
+    </section>
+    <section class="guests_section">
+      <h3 class="featured_guests">Featured Guests</h3>
+      <ul class="guests" id="guests">
+
+      </ul>
+    </section>
+  </main>
+  <section id="sponsor" class="partner_section">
+    <h3 class="partner">Partner</h3>
+    <div class="img_split">
+      <img class="partner_img" src="./pics/bull.png" alt="disney"><img class="partner_img" src="./pics/google.png"
+        alt="lamborghini"><img class="partner_img" src="./pics/dew.png" alt="mountain dew">
+
+      <img class="partner_img" src="./pics/globe.png" alt="globe"><img class="partner_img" src="./pics/cola.png"
+        alt="cola">
+    </div>
+
```

Comment:
* Hi @Lucas-Masaba So sorry this requested change wasn't asked by the previous reviewer the footer section for the home page is missing kindly add the footer section as it is in the original design 
* with text to the right and log to the left hand side 
![image](https://user-images.githubusercontent.com/52497246/142459566-7654aa3b-df78-4f85-ac3f-d131d6c4b7ab.png)

Reply:
Thank you for having an awesome eye for this issue. I have added the footer.